ga4gh/backend.py

```diff
@@ -176,6 +176,20 @@ def _getEnd(cls, readAlignment):
             len(readAlignment.alignedSequence))
 
 
+class ReadGroupSetIntervalIterator(ReadsIntervalIterator):
+    """
+    An interval iterator for reads, for an entire ReadGroupSet
+    """
+    def __init__(self, request, parentContainer, reference):
+        self._reference = reference
```

Comment:
Isn't this identical to ReadsIntervalIterator at the moment? If so, why implement these methods again?